coalib/parsing/FilterHelper.py

```diff
@@ -7,6 +7,22 @@ def is_valid_filter(filter):
 
 
 def apply_filter(filter_name, filter_args, all_bears=None):
+    """
+    Returns bears after filtering based on ``filter_args``. It returns
+    all bears if nothing is present in ``filter_args``.
+
+    :param filter_name:
+        Name of the filter.
+    :param filter_args:
+        Arguments of the filter to be passed in.
+        For example:
+        ``['c', 'java']``
+    :param all_bears:
+        A list of all bears, so the program knows whether to get all bears.
+        Default is set to None
```

Comment:
Missing full stop. Also, we can explain what happens if the param `all_bears` is `None`.